README.md

```diff
@@ -1,5 +1,345 @@
+API key allows API access to both published and unpublished content. The public
+key only allows access to published content. The Vercel production environment
+is configured with the value of the public API key so that it cannot
+accidentally access the unpublished content. The Vercel preview environment (as
+well as the local development environment) uses the preview API key so that the
+website team can preview draft content.
+
+Each website's publish function is connected to a webhook on Vercel. When this
+Vercel webhook is called, it does a preview deployment based on the current
+commit of the `develop` branch on GitHub.
+
+The Storyblok preview iframe is configured to show content previews against the
+Vercel URL that points to the latest build off the `develop` branch. This was
+done to help ensure that if there are any possible issues or conflicts between
+code changes and content changes, they will be caught early on. The Storyblok
+editor automatically passes cache-busting query string parameters (as in
+`?_storyblok=<timestamp>`) to the preview URL, so it should be okay to a use a
+constant URL base (rather than the always changing SHA-based Vercel preview
+URLs).
+
+There is a fair amount of code in the codebase that integrates with Storyblok.
+For example, there is a bridge that syncs the site with changes coming from the
+Storyblok editor.
+
+There is also some middleware that checks if the HTTP referrer is
+`https://app.storyblok.com/`. In that case, the code assumes that the end user
+is coming to the site from within the Storyblok iframe. So it puts the user in
+Next.js preview mode. When the user is in preview mode, the website shows a
+yellow banner at the top of the page, letting the user know that they are seeing
+both published and unpublished content on the site.
+
+### Vercel
+
+Each website corresponds to a separate project in Vercel, quansight-consulting
+and quansight-labs, respectively. However, because both projects are mapped to
+the same repository on GitHub, whenever a commit is made to the repo, whether
+the commit was just for the Consulting site or just for the Labs site, it
+triggers a preview deployment for both sites. Likewise, whenever a merge is made
+to the `main` branch, it triggers a production deployment to both websites.
+
+Each Vercel project has settings that allow it to integrate with Storyblok,
+Next.js, and GitHub. Each Vercel project has three separate environments:
+development, preview and production. The development and preview environments
+contains the preview key to Storyblok. The production environment contains the
+public key to Storyblok.
+
+Each Vercel project is also configured with a webhook that Storyblok uses to
+kick off a preview deploy of the `develop` branch whenever content is published
+in Storyblok.
+
+### Slack
+
+There is a Vercel app for Slack. It is set up to send deployment notifications
+to the `#vercel-notifications` channel.
```

Comment:
A decision needs to be made here. Should there be one Slack channel for both Vercel projects (quansight-consulting and quansight-labs) or two separate channels? I lean towards one channel for simplicity. On the other hand, there is some minor potential for confusion, for example, if changes in the Storyblok Consulting space are made at the same time as changes in the Storyblok Labs space. The Vercel Slack bot will tell you which project it deployed, but you have to know how to look carefully at the message that the Vercel bot sends.

Reply:
I lean towards one as well.

Reply:
I like one channel also. It shouldn't be hard to train on where to look, to tell which bot message has the "Visit" link you should click.